admin_guide/managing_projects.adoc

```diff
@@ -94,3 +101,40 @@ or:
 To request a new project, fill out the project request form located at
 https://internal.example.com/openshift-project-request.
 ----
+
+== Node Selectors
+
+Node selectors are used in conjunction with labels to control pod placement. You can assign a pod or set of pods to specific nodes, or limit a project to only run on a desired node.
+
+For example, if you have a OpenShift topology with multiple regions, node selectors help restrict specific OpenShift projects to only deploy pods into a specific region. 
+
+To create a project with a node selector, use the `--node-selector` option. For example, the following creates a new project named `MyProject`, but runs only on the nodes labeled `user-node` and `east`:
+
+====
+----
+$ oadm new-project MyProject --node-selector='type=user-node,zone=east'
```

Comment:
It's a minor thing, but I would rather say "region=east" than "zone" here. Typically you want your pods deployed in one region but across multiple zones.